common/src/transport_api/v0.rs

```diff
@@ -30,6 +30,8 @@ impl_message!(DestroyPool);
 impl_vector_request!(Pools, Pool);
 impl_message!(GetPools);
 
+impl_vector_request!(NvmeSubsystems, NvmeSubsystem);
```

Comment:
Should this not be in v1.rs?

Reply:
NvmeSubsystem/s  are both control plane  specific structs. This macro allows us to have vector representation of **NvmeSubsystem** struct. Vector object would be **NvmeSubsystems** .

We have saparate v1, v1 specs when grpc is between control plane and data plane.